src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -80,6 +81,9 @@ class QuantizationModifier(ScheduledModifier):
         exception. For compatibility with YAML serialization only.
     :param model_fuse_fn_kwargs: dictionary of keyword argument values to be passed
         to the model fusing function
+    :param quantize_embeddings: if True, will perform QAT on torch.nn.Embedding layers
+        using sparseml.pytorch.utils.quantization.prepare_embeddings_qat to fake
+        quantize embedding weights. Default is True
```

Comment:
Let's add a note in here that it won't do anything for models that don't contain embeddings since our default is True